examples/update-demo/v1beta3/kittern-rc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  name: update-demo-kitten
+spec:
+  replicas: 0
+  selector:
+    name: update-demo
+    version: kitten
+  template:
+    metadata:
+      labels:
+        name: update-demo
+        version: kitten
+    spec:
+      containers:
+      - capabilities: {}
+        image: kubernetes/update-demo:kitten
+        imagePullPolicy: IfNotPresent
+        name: update-demo
+        ports:
+        - containerPort: 80
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
```

Comment:
Please remove terminationMessagePath, since this container doesn't write that path.